<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>460</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>371</width>
      <height>431</height>
     </rect>
    </property>
    <widget class="QPushButton" name="btnSend">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>400</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSetUp">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>400</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Set up</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="tEditSend">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>360</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClearSend">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>330</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear Send</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>331</y>
       <width>291</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Send:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="tEditReceive">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>371</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnClearReceive">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>76</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear Receive</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnTest">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Test</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Receiveï¼š</string>
     </property>
    </widget>
    <widget class="QScrollBar" name="verticalScrollBar">
     <property name="geometry">
      <rect>
       <x>351</x>
       <y>39</y>
       <width>20</width>
       <height>281</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>311</width>
      <height>421</height>
     </rect>
    </property>
    <widget class="QPushButton" name="btnOpen">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>210</y>
       <width>75</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>291</width>
       <height>191</height>
      </rect>
     </property>
     <property name="title">
      <string>Port Set up</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>271</width>
        <height>164</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>COM-PORT</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cBoxComPort"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Baund rate:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="cBoxBaundRate"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Data bits:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="cBoxDataBits"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Parity bit:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="cBoxParityBit"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Stop bit:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="cBoxStopBit"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Flow control:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="cBoxFlowControl"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>291</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Send set</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QRadioButton" name="rBtnASCIISend">
         <property name="text">
          <string>ASCII</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rBtnHEXSend">
         <property name="text">
          <string>HEX</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>104</x>
        <y>25</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QCheckBox" name="checkBoxAutoResend">
         <property name="text">
          <string>Auto resend</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cBoxTimeMs"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>291</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Receive Set up</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>151</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="checkBoxWrapAuto">
         <property name="text">
          <string>Wrap auto</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxShowSend">
         <property name="text">
          <string>Show send</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxDisplayTime">
         <property name="text">
          <string>Display time</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>23</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="rBtnASCIIRec">
         <property name="text">
          <string>ASCII</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rBtnHEXRec">
         <property name="text">
          <string>HEX</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
